core/storage/learner_group/gae_models_test.py

```diff
@@ -0,0 +1,276 @@
+        expected_invited_user_data = {
+            '3232': {
+                'title': 'title',
+                'description': 'description',
+                'facilitator_user_id': '',
+                'student_user_id': '',
+                'invited_user_id': 'user_6',
+                'subtopic_page_ids': ['subtopic_1', 'subtopic_2'],
+                'story_ids': ['story_1', 'story_2']
+            }
+        }
+        self.assertEqual(expected_invited_user_data, invited_user_data)
+
+    def test_export_data_on_facilitators(self) -> None:
+        """Test export data on users that are facilitators of
+        the learner group.
+        """
+        facilitator_user_data = (
+            learner_group_models.LearnerGroupModel.export_data('user_1'))
+        expected_facilitator_user_data = {
+            '3232': {
+                'title': 'title',
+                'description': 'description',
+                'facilitator_user_id': 'user_1',
+                'student_user_id': '',
+                'invited_user_id': '',
+                'subtopic_page_ids': ['subtopic_1', 'subtopic_2'],
+                'story_ids': ['story_1', 'story_2']
+            }
+        }
+
+        self.assertEqual(
+            expected_facilitator_user_data,
+            facilitator_user_data)
+
+    def test_export_data_on_uninvolved_user(self) -> None:
+        """Test export data on users who do not have any involvement with
+        the learner group.
+        """
+        uninvolved_user_data = (
+            learner_group_models.LearnerGroupModel.export_data('user_31'))
+        expected_uninvolved_user_data: Dict[str, Union[str, List[str]]] = {}
+
+        self.assertEqual(
+            expected_uninvolved_user_data,
+            uninvolved_user_data)
+
+    def test_apply_deletion_policy_on_students(self) -> None:
+        """Test apply_deletion_policy on users that are students of
+        the learner group.
+        """
+        self.assertTrue(
+            learner_group_models.LearnerGroupModel
+            .has_reference_to_user_id('user_2'))
+
+        (
+            learner_group_models.LearnerGroupModel
+            .apply_deletion_policy('user_2')
+        )
+
+        self.assertFalse(
+            learner_group_models.LearnerGroupModel
+            .has_reference_to_user_id('user_2'))
+
+    def test_apply_deletion_policy_on_invited_users(self) -> None:
+        """Test apply_deletion_policy on users that have been
+        invited to join the learner group.
+        """
+        self.assertTrue(
+            learner_group_models.LearnerGroupModel
+            .has_reference_to_user_id('user_5'))
+
+        (
+            learner_group_models.LearnerGroupModel
+            .apply_deletion_policy('user_5')
+        )
+
+        self.assertFalse(
+            learner_group_models.LearnerGroupModel
+            .has_reference_to_user_id('user_5'))
+
+    def test_apply_deletion_policy_on_facilitators(self) -> None:
```

Comment:
Please add tests for this model to wipeout_service_test.py

Reply:
Done!